core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -754,6 +780,16 @@ void processMeasurementEvent(WriterJob job) {
                             break;
                         }
 
+                        case ColumnType.GEOSHORT:
```

Comment:
it may look inconsistent as far as case label naming goes, however a label is just that, I suppose I could have declared extra `NewLineProtoParser` "labels" but that would be a inconsistent with their intended use. The value on which the switch statement branches comes from `L550`, it is a storage tag, this is an optimisation. I will add a comment.